less/mixins.less

```diff
@@ -1 +1,35 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.button(@width, @bg-color){
+    width: @width;
+    background-color: @bg-color;
+    border-radius: 30px;
+    display: flex;
+}
```

Comment:
dope use for a mixin, it could be cool to add in the hover right to this.
then you can throw in some parameters for the hover and add the little extra to every button